src/validation.h

```diff
@@ -523,6 +521,14 @@ class CChainState {
      */
     CCriticalSection m_cs_chainstate;
 
+    /**
+     * Is this chainstate undergoing initial block download?
+     *
+     * Mutable because we need to be able to mark IsInitialBlockDownload()
+     * const, which toggles this for caching purposes.
```

Comment:
in commit 61fd34e refactoring: IsInitialBlockDownload -> CChainState

This is only toggled once (aka latched)? So should keep that nomenclature?